app/models/message.rb

```diff
@@ -97,6 +97,16 @@ def upvs_form
     )
   end
 
+  def update_html_visualization
+    self.update(
+      html_visualization: visualization
+    )
+
+    form.update(
+      visualizable: html_visualization.present?
+    )
+  end
+
   def visualization
```

Comment:
Ked uz sme tu, tak by som aj toto prepisal na build_visualization lebo to nie je getter

Reply:
Upravila som